modules/porous_flow/src/actions/PorousFlowAddMaterialJoiner.C

```diff
@@ -65,6 +66,16 @@ PorousFlowAddMaterialJoiner::act()
         // Check if the material is evaluated at the nodes or qps
         const bool at_nodes = params.get<bool>("at_nodes");
 
+        // Get the vector of blocks that the material is restricted to
+        const std::vector<SubdomainName> block_ids =
+            params.get<std::vector<SubdomainName>>("block");
+
+        // String version of block id to use to ensure unique names
+        std::string block_str;
```

Comment:
Did you want to have
```
std::string block_str = "";
```
or
```
const std::string block_str = (block_ids.empty() ? "" :  "_" + Moose::stringify(block_ids[0]));
```

Reply:
I forgot to take these out after realising that I could add a single PorousFlowJoiner material that combined all block restricted materials.